server/models/decorators/Fix.ts

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
This is the most unfortunate thing about this PR, all very hacky but hopefully it's somewhat temporary and a better solution will be available as the tooling matures.